.circleci/config.yml

```
@@ -0,0 +1,16 @@
+version: 2.1
+orbs:
+  architect: giantswarm/architect@0.1.2
+
+workflows:
+  package-and-push-chart-on-tag:
+    jobs:
+      - architect/push-to-app-catalog:
+          name: "package and push kong-app chart"
+          app_catalog: "giantswarm-incubator-catalog"
+          app_catalog_test: "giantswarm-incubator-test-catalog"
+          chart: "kong-app"
+          # Trigger job on git tag.
```

```suggestion
          # Needed to trigger job also on git tag.
```
The same sentence as `aws-operator`, `chart-operator`

---

Thanks, I took this from prometheus-operator-app I'll update there too.